app/assets/stylesheets/sufia/_styles.scss

```diff
@@ -75,3 +75,13 @@ label.disabled {
   text-decoration:underline;
   background-color:#0088CC;
 }
+
+.check_all {
+  white-space: nowrap;
+  .caret {
+    margin-left: 0;
+  }
+  input#check_all {
```

Comment:
Using `input` instead of `input#check_all` doesn't work.  Just `input` isn't specific enough to override a previous use of `input#check_all`